src/main/java/net/sf/jabref/logic/importer/fileformat/ModsImporter.java

```diff
@@ -218,7 +216,7 @@ private void parseIdentifier(Map<String, String> fields, IdentifierDefinition id
     }
 
     private void parseTopic(Map<String, String> fields, List<JAXBElement<?>> topicOrGeographicOrTemporal,
-            List<String> keywords) {
+                            List<String> keywords) {
```

Comment:
I'm not sure about our conventions regarding code indention. @koppor is this ok?